sklearn/metrics/cluster/_supervised.py

```diff
@@ -408,7 +407,7 @@ def adjusted_rand_score(labels_true, labels_pred):
         return 1.0
 
     return 2.0 * (tp * tn - fn * fp) / ((tp + fn) * (fn + tn) + (tp + fp) * (fp + tn))
-
+    
```

Comment:
There was an indent inserted into this line as well.

Reply:
Thank you @stefmolin